src/standalone/Vegetation/canopy_surface_fluxes.jl

```diff
@@ -0,0 +1,61 @@
+export MoninObukhovCanopyFluxes, MoninObukhovHeightBased
+abstract type AbstractCanopySFParameterization{FT <: AbstractFloat} end
+
+"""
+    MoninObukhovCanopyFluxes{FT, F <: Union{FT, ClimaCore.Fields.Field}} <: AbstractCanopySFParameterization{FT}
+
+A parameterization specifying how to compute latent and sensible heat
+fluxes between the atmosphere and the canopy based on Monin-Obukhov
+Surface Theory.
+
+You must specify
+- a minimum roughness length (global constant)
+- the roughness length for momentum (can be a constant or a field)
+- the roughness length for scalars (can be a constant or a field)
+- the displacement height (can be a constant or a field)
+- the parameter `η` which specifies how to upscale from leaf level
+  to canopy fluxes via the leaf boundary layer resistance.
+"""
+struct MoninObukhovCanopyFluxes{FT, F <: Union{FT, ClimaCore.Fields.Field}} <:
+       AbstractCanopySFParameterization{FT}
+    "Minimum roughness length (m)"
+    z_0min::FT
+    "Canopy roughness length for momentum (m)"
+    z_0m::F
+    "Canopy roughness length for scalars (m)"
+    z_0b::F
+    "Canopy displacement height (m)"
+    displ::F
+    "Constant used to compute the resistance for fluxes between the leaves and canopy airspace (unitless)"
+    η::FT
+end
+
+"""
+    MoninObukhovHeightBased(toml_dict, height)
+
+A constructor for a MoninObukhovCanopyFluxes surface flux theory,
+ specifying how to compute latent and sensible heat
+fluxes between the atmosphere and the canopy based on Monin-Obukhov
```

Comment:
also turbulent moisture flux and momentum fluxes right? Maybe we could just say "specifying how to compute turbulent fluxes between the atmosphere and canopy ..."